macos.go

```diff
@@ -124,6 +85,8 @@ func DeleteItemRef(ref C.CFTypeRef) error {
 var (
 	// KeychainKey is key for kSecUseKeychain
 	KeychainKey = attrKey(C.CFTypeRef(C.kSecUseKeychain))
+	// KeychainDataProtectionKey is key for kSecUseDataProtectionKeychain
+	KeychainDataProtectionKey = attrKey(C.CFTypeRef(C.kSecUseDataProtectionKeychain))
```

Comment:
kSecUseDataProtectionKeychain is only available in macOS 10.15, causing compilation error on CI. It might be worth adding that in a seperate PR?